src/www/ui/admin-dashboard.php

```diff
@@ -308,25 +352,43 @@ public function Output() {
     $this->pgVersion = pg_version($PG_CONN);
     
     $V="";
-    $V .= "<table border=0 width='100%'><tr>\n";
+    $V .= "<table style='width: 100%;' border=0>\n";
```

Comment:
Please do not use inline styling. Please use CSS instead of table specific styling options.